pages/linux/chcpu.md

```diff
@@ -0,0 +1,11 @@
+# chcpu
+
+> Enable/disable system's CPUs.
```

Comment:
Hey! I don't seem to have privileges to edit this PR, but perhaps this would read better?

```suggestion
> Enable/disable a system's CPUs.
```

Reply:
I'm sorry @sbrl! Changes are applied.

Reply:
np